packages/jaeger-ui-components/src/index.ts

```diff
@@ -1,6 +1,6 @@
 export { default as TraceTimelineViewer } from './TraceTimelineViewer';
 export { default as TracePageHeader } from './TracePageHeader';
-export { default as SpanBarSettings } from './settings/SpanBarSettings';
+export { default as SpanBarSettings } from '../../../public/app/core/components/SpanBarSettings/SpanBarSettings';
```

Comment:
Also this should change.

Reply:
I don't see `SpanBarSettings` defined anywhere else apart from in core so I'm not sure where else I can import it from?